src/exporters/http/retry.rs

```diff
@@ -83,14 +88,23 @@ impl<Resp> RetryPolicy<Resp> {
     }
 
     pub fn new(retry_config: RetryConfig, is_retryable: Option<RetryableFn<Resp>>) -> Self {
+        // We immediately drop the receiver channel and only keep receivers open for
+        // active retries
+        let (tx, _) = broadcast::channel(100);
```

Comment:
Is there any significance to capacity here? If so might be worth a comment specifying what it should be greater to. 0 will panic, but i.e. is 1 or 10 sufficient?

Reply:
Yeah, 100 was arbitrary, I've seen 16 in some examples which may be better. I think mostly > 1 is fine.